.retest/recheck.ignore.js

```diff
@@ -0,0 +1,46 @@
+var allowedPxDiff = 5;
+var baseUrl = /http[s]?:\/\/[\w.:\d\-]*/;
+var fontFamilies = [["system-ui", "Arial"], ["-apple-system", "sans-serif"]];
```

Comment:
Is the leading hyphen in `"-apple-system"` intended?

Reply:
Yes, this is how it shows up at my system...